src/seedu/addressbook/parser/Parser.java

```diff
@@ -248,5 +252,13 @@ private Command prepareFind(String args) {
         return new FindCommand(keywordSet);
     }
 
-
+    private Command preparePhone(String args) {
```

Comment:
Since the name of the method does not explicitly express the functionality of it, it would be better to have a header comment explaining what it does.